pkgs/development/python-modules/oracledb/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, cryptography
+, cython
+}:
+
+buildPythonPackage rec {
+  pname = "oracledb";
+  version = "1.2.2";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3Z9jCE5EZCtISkay/PtPySHzn6z0lKG6sAYo+mQJ9Pw=";
+  };
+
+  buildInputs = [
+    cython
+  ];
+
+  propagatedBuildInputs = [
+    cryptography
+  ];
+
+  doCheck = false;  # Checks need an Oracle database
+
+  pythonImportsCheck = [
+    "oracledb"
+  ];
+
+  meta = with lib; {
+    description = "Python driver for Oracle Database";
+    homepage = "https://oracle.github.io/python-oracledb";
```

Will allow some tools to show the changelog in the future.

```suggestion
    homepage = "https://oracle.github.io/python-oracledb";
    changelog = "https://github.com/oracle/python-oracledb/blob/v${version}/doc/src/release_notes.rst";
```